components/raftstore/src/store/fsm/apply.rs

```diff
@@ -572,6 +572,24 @@ fn should_write_to_engine(cmd: &RaftCmdRequest, kv_wb_keys: usize) -> bool {
     false
 }
 
+/// Checks if a write is needed to be issued after handling the command.
+fn should_write_to_engine_after(cmd: &RaftCmdRequest) -> bool {
```

Comment:
This seems to be completely different from `should_write_to_engine`. It just checks whether need to flush to disk. I suggest to name it as `should_sync_log`.

Reply:
Sure.